docs/installation/airgap.md

```diff
@@ -5,35 +5,54 @@ weight: 60
+are using the bundled containerd as the container runtime,
+and have a OCI-compliant private registry available in your environment.
 
-### Create the Registry YAML
+If you have not yet set up a private Docker registry, refer to the [official Registry documentation](https://docs.docker.com/registry/deploying/#run-an-externally-accessible-registry).
 
-Follow the [Private Registry Configuration](private-registry.md) guide to create and configure the registry.yaml file.
+#### Create the Registry YAML and Push Images
 
-Once you have completed this, you may now go to the [Install K3s](#install-k3s) section below.
+1. Obtain the images archive for your architecture from the [releases](https://github.com/k3s-io/k3s/releases) page for the version of K3s you will be running.
+2. Use `docker image load k3s-airgap-images-amd64.tar.zst` to import images from the tar file into docker.
+3. Use `docker tag` and `docker push` to retag and push the loaded images to your private registry.
+4. Follow the [Private Registry Configuration](private-registry.md) guide to create and configure the `registries.yaml` file.
+5. Proceed to the [Install K3s](#install-k3s) section below.
 
+### Manually Deploy Images Method
 
-## Manually Deploy Images Method
+These steps assume you have already created nodes in your air-gap environment,
+are using the bundled containerd as the container runtime,
+and cannot or do not want to use a private registry.
 
-We are assuming you have created your nodes in your air-gap environment and use containerd as container runtime.
-This method requires you to manually deploy the necessary images to each node and is appropriate for edge deployments where running a private registry is not practical.
+This method requires you to manually deploy the necessary images to each node, and is appropriate for edge deployments where running a private registry is not practical.
 
-### Prepare the Images Directory and K3s Binary
-Obtain the images tar file for your architecture from the [releases](https://github.com/k3s-io/k3s/releases) page for the version of K3s you will be running.
+#### Prepare the Images Directory and Airgap Image Tarball
 
-Place the tar file in the `images` directory, for example:
+1. Obtain the images archive for your architecture from the [releases](https://github.com/k3s-io/k3s/releases) page for the version of K3s you will be running.
+2. Download the imagess archive to the agent's images directory, for example:
+  ```bash
+  sudo mkdir -p /var/lib/rancher/k3s/agent/images/
+  sudo curl -L -O /var/lib/rancher/k3s/agent/images/k3s-airgap-images-amd64.tar.zst https://github.com/k3s-io/k3s/releases/download/v1.29.1-rc2%2Bk3s1/k3s-airgap-images-amd64.tar.zst
+  ```
+3. Proceed to the [Install K3s](#install-k3s) section below.
 
-```bash
-sudo mkdir -p /var/lib/rancher/k3s/agent/images/
-sudo cp ./k3s-airgap-images-$ARCH.tar /var/lib/rancher/k3s/agent/images/
-```
+### Embedded Registry Mirror
```

Comment:
Can we list this as "experimental" in the docs for now as well?

Reply:
I second this